dataplug-instagram/conf/evolutions/dataplug.sql

```diff
@@ -7,3 +7,9 @@ VALUES
   ('feed', 'User feed''s media', 'sequence'),
   ('profile', 'User''s Instagram profile information', 'snapshots')
   ON CONFLICT (name) DO NOTHING;
+
+--changeset dataplugInstagram:deleteEndpoints context:data
+
```

Comment:
Assuming Instagram authentication tokens and/or user IDs have changed as well, we should clean up the authentication token table as well.